templates/data/kafka/managed.html

```diff
@@ -0,0 +1,330 @@
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Key features</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">Backed by an SLA</h3>
+            Backed by a 99.9% uptime SLA. Your data is the backbone of your business. Canonical ensures that it's available with a high uptime commitment. Canonical managed services for Kafka keep the data flowing.
+          </li>
+          <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">Secure by design</h3>
+            Service delivery with security in mind. Minimise risks. Canonical managed services are routinely audited for conformance to industry standards and legislation like GDPR, ISO:27001 and SOC-2 Type II.
+          </li>
+          <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">Turnkey service</h3>
+            A complete solution, on your cloud tenancy or in your data centre. Canonical covers the full project engagement including architecture design, deployment, transition and day to day operations.
+          </li>
+          <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">Service health dashboards</h3>
+            Service health dashboards for full transparency. We provide you with access to Kafka health dashboards, so that you can verify the health and uptime of your service directly.
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Managed Kafka pricing</h2>
+      </div>
+      <div class="col">
+        <p>A low total cost of ownership. Up to five times lower than cloud services when fully costed.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <hr class="p-rule--highlight u-no-margin--bottom" />
+        <div class="p-card--overlay u-no-padding--top" style="padding: 1.5rem;">
+          <div class="row p-section--shallow">
+            <div class="col-3 col-medium-3 col-small-2">
+              <h3>Managed Kafka on Azure</h3>
+            </div>
+            <div class="col-3 col-medium-3 col-small-2">
+              <p class="p-heading--3 u-align-text--right u-no-margin--bottom">$3099</p>
+              <p class="u-text--muted u-align-text--right u-no-margin--bottom">
+                Per year,
+                <br />
+                per VM <span aria-labelledby="Management fee does include the cloud infrastructure cost.">*</span>
+              </p>
+            </div>
+          </div>
+          <div class="p-section--shallow">
```

Comment:
There is no shallow section here on design

Reply:
It may not be explicitly declared, but this spacing has to come from somewhere. This bottom spacing isn't the default for p elements 
![image](https://github.com/user-attachments/assets/f9ba0063-01fa-4a1a-83bd-44e471d1adf5)